cmd/osde2ectl/delete/cmd.go

```diff
@@ -132,6 +132,8 @@ func run(cmd *cobra.Command, argv []string) error {
 			}
 		}
 		return allErrors.ErrorOrNil()
+	} else {
```

Comment:
This won't catch it correctly. You should if/else the cluster_id check explicitly, and change the `if owner` block to be its own individual if-statement. :)

Reply:
i am dumb need more coffee